gensim/models/keyedvectors.py

```diff
@@ -154,6 +154,19 @@ def get_vector(self, entity):
         else:
             raise KeyError("'%s' not in vocabulary" % entity)
 
+    def add(self, entity, weights):
```

Comment:
What's about re-using this function in  (this is duplication right now from https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/utils_any2vec.py#L182)?

Reply:
Also, need to add tests to check this functionality:
- load some kv, add more vectors in a manual way and check that this added fine
- create empty kv, fill it manually and check that all fine

Reply:
@menshikh-iv 

About reusing this function:

It's a bit difficult because in this function `vstack` is used to append new word vector to `self.vectors`, while `add_word` in utils_any2vec creates `vectors` array at first and then it just inserts vectors into zeros array (https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/utils_any2vec.py#L197).

While it's possible to follow DRY here, the interface of `BaseKeyedVectors.add()` method will be more complicated (or I can change the logic in `utils_any2vec` -- not to create `vectors = np.zeros(...)` but append each word to the array, but it could decrease the performance of `load_word2vec_format` function). 

If some of these two options is okay, I'll implement it.

Reply:
Aha, thanks for suggestion, let's stay it as is.